applications/skyepass.md

```diff
@@ -185,32 +185,30 @@ Besides private work for companies that cannot be shared, Song developed a simpl
-| 1.     | Desktop App/Browser Extension  | **Desktop App**<br/>A React.js + Electron App to start with. The App will implement as close as possible to the graphic design (per 3). <br/>- Wallet Creation / Backup Phase / Create Master Password<br/>- Wallet Import / Signin<br/>- Autolock after timed inactivity or manually lock the App <br/>- Create/Share/Manage Vaults<br/>- Add/Update/Delete Password Items (with 2FA OTP support)<br/>- Add/Update/Delete Secure Note/Credit Card <br/>- Basic ETH wallet extension<br/>- Basic Polkadot wallet extension <br/> - Application Marketplace<br/>- sharing a single password item directly to another user<br/><br/>**Browser Extension** <br/>- Communication to Desktop Application<br/>- Auto-fill account/passwords<br/> <br/>**IPFS Connection**<br/>- upload / update / read from a public IPFS node. |
-| 2.     | Smart Contract                 | Finish all 4 smart contracts mentioned in the `Ecosystem Fit` section. For this phase, these contracts are testing contracts, and they are subject to minor changes as long as they provide the same/very similar functionality. We might use the "identity pallet" instead of deploying smart contract 3 & 4. We will use Rust, ink! for smart contract development, along with tools like Redspot by [Patract Hub](https://patract.io/?lang=en-US) |
-| 3.     | UI/UX Design Standards         | Color Schema, Composition & Balance, Typography, Animation, Navigation, Icon Set, Brand Logo |
-| 4.     | Basic Marketing Efforts        | send out private alpha invitation                            |
-| 5.     | Testing                        | Unit Testing                                                 |
-### Milestone 2  — Ready for Beta Release
+| 0b.    | Documentations/Project Website | A minimalism project homepage with a user manual that documents how to use SkyePass from account creation to manage passwords, and install extensions and browser extensions. <br/>A developer documentation that describes the encryption mechanism in details; |
+| 1.     | Core Encryption Library        | Refined Core Encryption API to deliver three core APIs: `upstream`, `downstream` and `updateEncryptionSchema`. <br/>The encryption library should be able to: <br/>1. Post files onto an Infura IPFS pinning service<br/>2. Definitive length for metadata<br/>3. Able to interact with Substrate WASM smart contracts and a decentralized storage network.<br/>4. Reusable by other products other than SkyePass |
+| 2.     | Smart Contract                 | Deploy a smart contract for access control and file coordination. We will use Rust, ink! for smart contract development, along with tools like Redspot by [Patract Hub](https://patract.io/?lang=en-US) |
```

Comment:
Why did you switch from implementing all 4 smart contracts under 'Ecosystem Fit' to just one?

Reply:
Explained in the email. TL;DR version for that is we figured out another (a much better) way to deliver the identity management capacity needed and we are currently working with the on-chain treasury team on it. All things related to identity can be removed from the application now.

Reply:
Also, have fixed the JSON thing in the most recent push. My editor was acting stupid :(